gosnmp.go

```diff
@@ -78,6 +75,9 @@ type GoSNMP struct {
 	// Conn is net connection to use, typically establised using GoSNMP.Connect()
 	Conn net.Conn
 
+	// MaxOids is the maximum number of oids allowed in a Get()
+	MaxOids int
```

Comment:
This breaks old code. **MaxOids** should default to **maxOids**, but how to implement?

Setting it in **Default** on line 98 isn't sufficient, as existing/new implementors mighn't be using **Default** -- structs should have useful "zero values". See http://dave.cheney.net/2013/01/19/what-is-the-zero-value-and-why-is-it-useful



A